src/github_test.ts

```diff
@@ -117,3 +121,26 @@ Deno.test("fetchPrFileNames() can handle big PRs", async () => {
   const aPrWith669Files = await fetchPrFileNames(24147);
   assertEquals(aPrWith669Files.size, 669);
 });
+
+Deno.test("fetchLastComment() returns the appropriate comment", async () => {
+  const prToLastComment = {
+    10: "Closing as fixed by #199 ",
+    29: null,
+    1000: "LGTM",
+    10000:
+      "It is a feature of SQL databases. The repo id is stored in the database and uses auto increment:\r\nhttps://www.w3schools.com/sql/sql_autoincrement.asp",
+  };
+  await Promise.all(
+    Object.entries(prToLastComment).map(
```

Comment:
```suggestion
    Object.entries(prToLastComment).forEach(
```

don't use `map` if you are not using the return value.

Reply:
Actually, it is used, retracting this :)